ods_ci/tests/Tests/0100__platform/0102__monitor_and_manage/0102__alerts/0102__alerts.robot

```diff
@@ -334,6 +334,37 @@ Verify Data Science Pipelines Application Alerts
 
     [Teardown]    Delete Project Via CLI    ${PROJECT}
 
+
+Verify Data Science Pipelines Operator Alert Fires When Operator Is Down
+    [Documentation]     Verifies that alert "Data Science Pipelines Operator Probe Success Burn Rate (for 5m)" is fired
+    [Tags]    Tier3
+    ...       ODS-2166
+    ...       RHOAIENG-13262
+    ...       Monitoring
+    Skip If Alert Is Already Firing    ${RHODS_PROMETHEUS_URL}
```

Comment:
I know we already have this in other test cases but do we want to skip this gracefully if an alert is already firing? I would let it run and possibly fail so we know the cluster wasn't clean/RHOAI wasn't correctly installed.

Reply:
Good point, removed here. Should I also remove it from the other tests?

Reply:
I would say so. Thanks!